erpnext/accounts/report/cash_flow/cash_flow.py

```diff
@@ -135,14 +135,21 @@ def get_cash_flow_accounts():
 		"section_name": "Investing",
 		"section_footer": _("Net Cash from Investing"),
 		"section_header": _("Cash Flow from Investing"),
-		"account_types": [{"account_type": "Fixed Asset", "label": _("Net Change in Fixed Asset")}],
+		"account_types": [
+			{"account_type": "Current Asset", "label": _("Net Change in Current Assets")},
```

Comment:
Current asset and current liability should form part of operating activities.
https://www.wallstreetprep.com/knowledge/change-in-net-working-capital-nwc/

Reply:
The article also reflecta my previous understanding of the two concepts:

> Current Liabilities

"Within the year"

> Current Assets

"Within the year"

The article goes on to say:

> the more practical formula excludes cash and short-term investments like marketable securities and commercial paper, as well as any interest-bearing debt such as loans and bonds.

And goes on to define more narrow concepts:

> Operating Current Assets
> Accounts Receivables (A/R), Inventory, Prepaid Expenses

or: Current Assets - Marketeable Securities

> Operating Current Liabilities
> Accounts Payable (A/P), Accrued Expense

or: Current Assets - Interest bearing debt

Reply:
The problem at hand appears to be that "Current" ("within the year") may contain proper financing and proper operating activities.

Reply:
Because this distinction matters quite a bit to a CFO, I'd propose to include another type "Current Operating Assets" and "Current Operating Liabilities" to disambiguate and capture anything not already covered by ome of the other types, but still residing within the operating category.

Reply:
We already have plans to refactor account type.
You can share your inputs [here](https://github.com/frappe/erpnext/issues/39923#issuecomment-2230045374).

Agree about the importance of distinction between operating and non-operating current assets. However, this is more inclined towards operating and non-operating is more likely expected to be insignificant.

Reply:
> Also, non-operating is more likely expected to be insignificant.

For one, we have short term interest bearing loans / credit lines, which are extremely important to manage carefully. But if that's a special case, I'm also fine with maintaining a local patch.

Let me come back with a conclusive change-set (to this PR) after thinking this over for a day or two.

> We already have plans to refactor account type.

That is _very_ nice to hear! I'll take some time and read through it.